random_question/weighted_random.py

```diff
@@ -10,11 +10,26 @@
 
 # TODO: Hess should implement!
 def weighted_sampler(data, weights):
-    """Should return a random item from data array, weighted by the weights array"""
```

Comment:
This comment is a `docstring`. that's how you write documentation in python. you shouldn't have deleted it :)

Reply:
a `docstring` is a triple-quote string at the top of a method/class

Reply:
done